src/Customer.js

```diff
@@ -1,3 +1,69 @@
-class Customer {}
+class Customer {
+  constructor(customerData, bookingData) {
+    this.id = customerData.id;
+    this.name = customerData.name;
+    this.bookings = [];
+
+    this.getBookings(bookingData);
+  }
+
+  getBookings(bookingData) {
+    const matchedBookings = bookingData.filter(booking => booking.userID === this.id);
+    this.bookings = matchedBookings;
+  }
+
+  sortBookings() {
+    return this.bookings.sort((a, b) => new Date(a.date) - new Date(b.date));
+  }
```

Comment:
dunno if you care - but I'm trying to do a past bookings a future bookings... now that i think more about it sounds hard